coalib/output/ConsoleInteraction.py

```diff
@@ -566,21 +566,31 @@ def choose_action(console_printer, actions):
     :return:                Return choice of action of user.
     """
     while True:
+        possible_actions = []
         console_printer.print(format_lines('(D)o nothing'))
+        possible_actions.append('D')
         for i, action in enumerate(actions, 1):
-            console_printer.print(format_lines('{:>2}: {}'.format(
-                i,
+            console_printer.print(format_lines('{}'.format(
                 action.desc)))
-
+            if action.desc == '(O)pen file':
```

Comment:
Hmm the idea is that in the end, the function needs to return an index based on what user choosed from the available actions and I don't think I can do that if I have all actions in the dict, we don't know the order of the actions. We can use a dict to just store the keys but I think it will be to much to use a dict for that. I think I can use just add instead of insert for optimization.